core/server/models/base/index.js

```diff
@@ -355,14 +362,21 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
         }
 
         return itemCollection.fetchPage(options).then(function formatResponse(response) {
-            var data = {};
+            options.columns = allColumns;
```

Comment:
suggestion:
now that we extend `options.columns`, maybe we can rename `allColumns` to `requestedColumns` (or something similar)